src/Interpreters/MySQL/InterpretersMySQLDDLQuery.cpp

```diff
@@ -117,6 +117,46 @@ static inline NamesAndTypesList getColumnsList(ASTExpressionList * columns_defin
     return columns_name_and_type;
 }
 
+static inline ColumnsDescription getColumnsDescription(NamesAndTypesList columns_name_and_type, ASTExpressionList * columns_define)
```

Comment:
I don't think this function should be 'inline'.

Reply:
Yea, I agree. Also removed `inline` from two massive functions that existed before.